content/SCALE/SCALETutorials/Apps/CommunityApps/TFTP-HPAApp.md

```diff
@@ -1,85 +1,114 @@
 ---
-title: "TFTPD-HPA"
-description: "Provides instructions for users migrating from the SCALE TFTP service to the new tftpd-hpa application." 
+title: "TFTP Server App"
+description: "Provides instructions for installing the TFTP Server application." 
 weight:
 aliases:
  - /scale/scaletutorials/apps/tftp-hpaapp/
+ - /scale/scaletutorials/systemsettings/services/tftpservicescale/
+ - /scale/scaleuireference/systemsettings/services/tftpservicescreen/
 tags:
 - scaletftp
 ---
 
-This article provides instructions on installing the new **tftpd-hpa** application.
-It also provides instructions on migrating from the deprecated SCALE 22.12.3 TFTP service to the tftpd-hpa application.
+The new **TFTP Sever** application provides Trivial File Transfer Protocol (TFTP) server functions.
+The TFTP Sever application is a lightweight TFTP-server container TrueNAS SCALE. It is not intended for use as a standalone container.
 
-The tftpd-hpa application is a lightweight TFTP-server container TrueNAS SCALE uses as a replacement for the SCALE service. 
-It is not intended for use as a standalone container.
+The app runs as root and drops privileges to the tftp (9069) user for the TFTP service. 
+Every appliation start launches a container with root privileges. 
+This checks the parent directory permissions and ownership.
+If it finds a mismatch, the container applies the correct permissions to the TFTP directories.
+If **Allow Create** is selected, the container also checks and chmods TFTP directories to 757 or to 555 if not checked. 
+Afterwards, the TFTP container runs as root user, dropping privileges to the tftp (9069) user for the TFTP service.
 
-## Before You Begin
-Before you configure the new tftpd-hpa application:
+## First Steps
 
-* Disable the TFTP service.
-  Go to **System Settings > Services** and disable the service and clear the **Start Automatically** checkbox. 
-  This prevents the service from re-enabling after a system restart.
-
-* Review your TFTP service settings and note all directory, host, auxiliary parameter, permission, and credential (username and password) settings.
+Configure DHCP server for network boot to work.
```

Comment:
This sentence feels a little vague and assumes a certain level of user knowledge, though I see why you wouldn't want to go into too much detail

Reply:
we can't tell them how to configure their DHCP server because we don't know their network setup. We need to be general here. But I'll make a small change to this sentence in my next commit.